src/cljam/io/cram.clj

```diff
@@ -0,0 +1,37 @@
+(ns cljam.io.cram
+  "Alpha - subject to change. Provides functions for reading from a CRAM file."
+  (:require [cljam.io.cram.core :as cram]
+            [cljam.io.protocols :as protocols]
+            [cljam.io.util :as io-util])
+  (:import [cljam.io.cram.reader CRAMReader]))
+
+(defn reader
+  "Creates a CRAM reader depending on the argument f: If f is a file or a string
+  that representing the path to a CRAM file, returns a new reader that reads
+  that CRAM file. If f is a CRAM reader, creates and returns a cloned CRAM reader
+  from it.
+
+  The function also takes an optional argument `option`, which is a map that
+  consists of:
+    - reference: a string representing the path to a reference file"
```

Comment:
I think it would be beneficial to document that an instance of `ISequenceReader` can also be passed, not just a string.

Reply:
Will do 💪